.github/workflows/coverity.yml

```diff
@@ -0,0 +1,25 @@
+name: coverity-scan
```

Comment:
`Coverity Scan` would look better in the GitHub UI.

Reply:
Makes sense.